styles/books/neuroscience/book.scss

```diff
@@ -131,6 +131,14 @@ $ChapterIntroType: side;
 // TOC
 @include use('TOC', 'carnival:::NoUnitToc');
 
+// EOB
+@include add_settings((
+  AppendixGlossary: (
+    _selectors: (".appendix > .column-container"),
+    'H3AppendixGlossaryTitle:::color': (_ref: "colorMap:::neuroscienceHeaderColor"),
+    'AppendixGlossaryContainer:::column-count': 2,
+  ),
+));
```

Comment:
We are keeping all the settings together on the top and all `use` methods at the bottom. Would you mind to move this over `@include use('BookRoot', "common:::BookRoot");` (L: 128)?